Cargo.toml

```diff
@@ -18,7 +18,7 @@ default = [
 ]
 
 [dependencies]
-bevy = {git = "https://github.com/bevyengine/bevy", rev = "7628f4a64e", default-features = false}
+bevy = {git = "https://github.com/bevyengine/bevy", version = "0.3.0", default-features = false}
```

Comment:
Maybe this should update the rev to avoid issues like #4 and because it records which bevy commit the plugin is tracking. For users who are on different but compatible git revs, the patch section can be used to override https://doc.rust-lang.org/cargo/reference/overriding-dependencies.html#the-patch-section.

Alternatively (or in addition), `bevy_webgl2` could check in a `Cargo.lock`.